src/auth/controllers/auth.controller.ts

```diff
@@ -51,11 +58,17 @@ export class AuthController {
   @UseGuards(LocalAuthGuard)
   @UseInterceptors(ClassSerializerInterceptor)
   login(
+    @ReqContext() ctx: RequestContext,
     @Req() req: Request,
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
     @Body() credential: LoginInput,
   ): BaseApiResponse<AuthTokenOutput> {
-    const authToken = this.authService.login(req.user as UserAccessTokenClaims);
+    this.logger.logWithContext(ctx, `${this.login.name} was called`);
+
+    const authToken = this.authService.login(
+      ctx,
+      req.user as UserAccessTokenClaims,
```

Comment:
This might become redundant now. Since `ctx` already has the `user` object in it.